diff --git a/browser/moz.build b/browser/moz.build
index 5a03cfc6d3f8fe3a677d4d9e9894e69a98d0c01f..516013c0f2392820a19e789f3f3d0e18fd83159e 100644
--- a/browser/moz.build
+++ b/browser/moz.build
@@ -53,6 +53,7 @@ if CONFIG["MOZ_ARTIFACT_BUILDS"]:
 # this directory, which is un-set in browser/app.
 JS_PREFERENCE_PP_FILES += [
     "app/profile/firefox.js",
+    "app/profile/aurelia.js",
 ]
 FINAL_TARGET_FILES.defaults += ["app/permissions"]
 
