diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -263,7 +263,8 @@
 #endif

 ; [Default Preferences]
 ; All the pref files must be part of base to prevent migration bugs
 @RESPATH@/browser/@PREF_DIR@/firefox.js
 @RESPATH@/browser/@PREF_DIR@/firefox-branding.js
+@RESPATH@/browser/@PREF_DIR@/aurelia.js
 @RESPATH@/greprefs.js
@@ -455,7 +456,5 @@
 #if defined(DESKTOP_LAUNCHER_ENABLED)
 @BINPATH@/desktop-launcher/desktop-launcher@BIN_SUFFIX@
 #endif

-#if defined(BUILT_BY_MOZILLA)
 @RESPATH@/distribution/*
-#endif
